content/en/ecosystem/vendors.md

```diff
@@ -43,6 +43,7 @@ OpenTelemetry in their commercial products.
 | Splunk                     | Yes               | Yes         | [splunk.com/blog/...](https://www.splunk.com/en_us/blog/conf-splunklive/announcing-native-opentelemetry-support-in-splunk-apm.html)                   |
 | Sumo Logic                 | Yes               | Yes         | [help.sumologic.com/](https://help.sumologic.com/docs/apm/traces/quickstart/)                                                                         |
 | TelemetryHub               | No                | Yes         | [telemetryhub.com](https://app.telemetryhub.com/docs)                                                                                                 |
+| Traceloop                  | Yes               | Yes         | [traceloop.dev](https://docs.traceloop.dev)                                                                                                 |
```

Comment:
Hmmm, does this count as a Yes for a [distribution](https://opentelemetry.io/docs/concepts/distributions/#what-is-a-distribution)?

Reply:
Specifically I was referring to https://github.com/traceloop/jest-opentelemetry/tree/main/packages/instrument-opentelemetry
Which we use so that we have more data that we can use in our tests (like HTTP body and headers). Does it count under "Additional capabilities beyond what OpenTelemetry provides"?

Reply:
It seems more like an instrumentation library + preconfigured otel exporter than a distribution. Typically distros wrap an upstream SDK rather than initialize the upstream SDK and enrich spans. Although the lines are pretty blurry.

Reply:
Makes sense :) changed to "no" then.